guide/02-api-overview/release-notes.ipynb

```diff
@@ -13,6 +13,18 @@
    "source": [
```

Comment:
<p>https://github.com/ArcGIS/geosaurus/pull/7674</p><p>arcgis.raster.ImageryLayer</p><p> </p><ol><li>Fix issues in saving an ImageryLayer item as a portal item with on the fly processing by setting for_viz is set to True.</li><li>Fix to honor the rendering rule and mosaic rule set on a portal item when initialising the imagery layer object.</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Added them!</p>